04.ls/ls.rb

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+COLUMN_COUNT = 3
+
+def main
+  filenames = Dir.glob('*')
+  show_file_list(filenames)
+end
+
+def show_file_list(filenames)
+  max_length = filenames.max_by(&:length).length
+
+  nested_file_names = convert_nested_filenames(filenames)
+  nested_file_names.each do |file_names|
+    file_names.each do |file_name|
+      print file_name.ljust(max_length + 2) unless file_name.nil?
+    end
+    puts
+  end
+end
+
+def convert_nested_filenames(filenames)
```

Comment:
convert Aだと「Aを変換する」の意味になります。
なので、この場合は「ネストしたファイル名の配列 "を" 変換する」という意味になります。

ですが、メソッドがやってることは「（ネストしていない）ファイル名の配列を、ネストした配列 "に" 変換する」処理です。
その場合、英語的には "convert filenames to/into nested filenames" が正です。
参考 https://ejje.weblio.jp/content/convert

ただ、何を変換するのかは引数で表現されているので、ここでは

```suggestion
def convert_to_nested_filenames(filenames)
```

という名前にするのがいいんじゃないかなーと思います。